cmd/discriminator/main.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
Is the cmd/discriminator a test script of sorts? It makes sense as far as showcasing the interaction JSON structure goes but I can't see much other than that in it, so I may be missing something.

Reply:
Correct, it's now been removed